tox.ini

```diff
@@ -1,6 +1,6 @@
 [tox]
 envlist =
-    py27,py34,py35,py36,cover,docs
+    py27,py34,py35,py36,py37,py38,cover,docs
```

Comment:
I think I'd rather not require developers to have a `python3.8` yet -- the Travis bits should be enough until there is an actual release to test against.

Reply:
Fixed in ae2c0defea334342f87b97418a028ea8785bc400. Thank you!